lineman/app/models/discussion_model.coffee

```diff
@@ -109,7 +109,7 @@ angular.module('loomioApp').factory 'DiscussionModel', (BaseModel, AppConfig) ->
       @remote.patchMember @keyOrId(), 'set_volume', { volume: volume }
 
     toggleStar: ->
-      @remote.patchMember @keyOrId(), if @starred then 'unstar' else 'star'
```

Comment:
Oof, I don't like doing it this way; it makes the toggleStar method counterintuitive and means we can't manually call it without first setting the opposite star value. I assume it's doing this because ng-change is firing the model change before this method, could we try something in the ng-model-options perhaps to change this?

```
input{ng-model: 'thread.starred', ng-model-options: '{updateOn: "blur"}'}
```

If blur doesn't work, I wonder if click, change, or mouseup might.